app/src/pages/api/business/documents/queries/getBusinessCampaigns.graphql

```diff
@@ -0,0 +1,25 @@
+query GetBusinessCampaigns {
+  getBusinessCampaigns {
+    id
+    slug
+    totalSatsInvested
+    investmentMultiple
+    btcPayServerStoreId
+    content {
+      asHtmlString
+      category
+      country
+      description
+      establishedAt
+      instagram
+      latitude
+      longitude
+      markerIcon
+      title
+      website
+      media {
+        featuredImageUrl
+      }
+    }
```

Comment:
Let's make this a fragment: `fragment BusinessCampaignContent on Business`

https://www.apollographql.com/docs/react/data/fragments/

Reply:
This is helpful when typing the return type of a given query or mutation hook result